wagtail/admin/viewsets/__init__.py

```diff
@@ -10,13 +11,17 @@ def __init__(self):
     def populate(self):
         for fn in hooks.get_hooks("register_admin_viewset"):
             viewset = fn()
-            if isinstance(viewset, (list, tuple)):
-                for vs in viewset:
-                    self.register(vs)
-            else:
-                self.register(viewset)
```

Comment:
This was added in 9d1110d349a43205407df0b5e73f33bd0a3a031e, but it's no longer used since the viewset registration was moved to `register_snippet` in 6e1e0240ef3ada234f6ee9d7db2584c141d29f78.